Crash when using `gtksignal = changed` with numeric textbox

```
using Gtk, GtkObservables
win = GtkWindow("testwin")
txt = textbox(5; gtksignal = "changed")
push!(win, txt)
showall(win)
```
Backspace to blank the textbox
```
┌ Warning: Executing #43:
└ @ Gtk.GLib C:\Users\ncb20\.julia\packages\Gtk\Xtubf\src\GLib\signals.jl:106
ERROR: AssertionError: xor(prev, current_task() !== g_stack)
Stacktrace:
  [1] g_siginterruptible(f::Gtk.GLib.var"#90#92"{Ptr{Gtk.GLib.GValue}, UInt32, Ptr{Gtk.GLib.GValue}, Vector{Any}, Bool, GtkObservables.var"#43#45"{Nothing, Observable{Int64}}, Ptr{Any}}, cb::Any)
    @ Gtk.GLib C:\Users\ncb20\.julia\packages\Gtk\Xtubf\src\GLib\signals.jl:209
  [2] GClosureMarshal(closuref::Ptr{Nothing}, return_value::Ptr{Gtk.GLib.GValue}, n_param_values::UInt32, param_values::Ptr{Gtk.GLib.GValue}, invocation_hint::Ptr{Nothing}, marshal_data::Ptr{Nothing})
    @ Gtk.GLib C:\Users\ncb20\.julia\packages\Gtk\Xtubf\src\GLib\signals.jl:58
  [3] set_gtk_property!(w::GtkEntryLeaf, name::String, value::String)
    @ Gtk.GLib C:\Users\ncb20\.julia\packages\Gtk\Xtubf\src\GLib\gvalues.jl:177
  [4] entrysetter!
    @ c:\users\ncb20\source\repos\GtkObservables.jl\src\widgets.jl:496 [inlined]
  [5] entrygetter(w::GtkEntryLeaf, observable::Observable{Int64}, range::Nothing)
    @ GtkObservables c:\users\ncb20\source\repos\GtkObservables.jl\src\widgets.jl:480
  [6] (::GtkObservables.var"#43#45"{Nothing, Observable{Int64}})(w::GtkEntryLeaf)
    @ GtkObservables c:\users\ncb20\source\repos\GtkObservables.jl\src\widgets.jl:446
  [7] (::Gtk.GLib.var"#90#92"{Ptr{Gtk.GLib.GValue}, UInt32, Ptr{Gtk.GLib.GValue}, Vector{Any}, Bool, GtkObservables.var"#43#45"{Nothing, Observable{Int64}}, Ptr{Any}})()
    @ Gtk.GLib C:\Users\ncb20\.julia\packages\Gtk\Xtubf\src\GLib\signals.jl:87
  [8] g_siginterruptible(f::Gtk.GLib.var"#90#92"{Ptr{Gtk.GLib.GValue}, UInt32, Ptr{Gtk.GLib.GValue}, Vector{Any}, Bool, GtkObservables.var"#43#45"{Nothing, Observable{Int64}}, Ptr{Any}}, cb::Any)
    @ Gtk.GLib C:\Users\ncb20\.julia\packages\Gtk\Xtubf\src\GLib\signals.jl:216
  [9] GClosureMarshal(closuref::Ptr{Nothing}, return_value::Ptr{Gtk.GLib.GValue}, n_param_values::UInt32, param_values::Ptr{Gtk.GLib.GValue}, invocation_hint::Ptr{Nothing}, marshal_data::Ptr{Nothing})
    @ Gtk.GLib C:\Users\ncb20\.julia\packages\Gtk\Xtubf\src\GLib\signals.jl:58
 [10] (::Gtk.var"#249#250")()
    @ Gtk C:\Users\ncb20\.julia\packages\Gtk\Xtubf\src\events.jl:2
 [11] g_sigatom(f::Any)
    @ Gtk.GLib C:\Users\ncb20\.julia\packages\Gtk\Xtubf\src\GLib\signals.jl:176
 [12] gtk_main()
    @ Gtk C:\Users\ncb20\.julia\packages\Gtk\Xtubf\src\events.jl:1
```

In my non-test code, I actually had a more serious error:
```
**
GLib-GObject:ERROR:../gobject/gsignal.c:882:emission_pop: code should not be reached
Bail out! GLib-GObject:ERROR:../gobject/gsignal.c:882:emission_pop: code should not be reached

Please submit a bug report with steps to reproduce this fault, and any error messages that follow (in their entirety). Thanks.
Exception: EXCEPTION_BREAKPOINT at 0x7ffa04c790d2 -- DebugBreak at C:\Windows\System32\KERNELBASE.dll (unknown line)
in expression starting at none:0
DebugBreak at C:\Windows\System32\KERNELBASE.dll (unknown line)
Allocations: 44873828 (Pool: 44858983; Big: 14845); GC: 39
```

--- a/src/widgets.jl
+++ b/src/widgets.jl
@@ -371,6 +371,9 @@ function textbox(::Type{T};
     if T <: AbstractString && range !== nothing
         throw(ArgumentError("You cannot set a range on a string textbox"))
     end
+    if T <: Number && gtksignal == "changed"
+        throw(ArgumentError("The `changed` signal with a numeric textbox is not supported."))
+    end
     obsin = observable
     observable, value = init_wobsval(T, observable, value; default="")
     if own === nothing

